### ISSUE TITLE
Awards Voting Results Export

### ISSUE THREAD

**Issue by jubjamie** [backstagetechnicalservices/website#17]
It would be handy next year to have an export of the votes for use at the BBQ.  Current option is to copy and paste the contents of the scrolling div into word and do some formatting. Can't print the page either as it's all in a scrolled div or similar.

Cheers

**Comment by bnjns** [comment]
Yeah, I thought about this earlier, too. Would you want just an easier way of seeing all of the winners, or would the ability to just print off the winners also work?

**Comment by jubjamie** [comment]
I think the lowest effort way would be too just have a page that is the results but not bounded by scroll and with a white background. Then you can just print that off.
